src/omero_zarr/raw_pixels.py

```diff
@@ -91,6 +91,12 @@ def planeGen() -> np.ndarray:
         cache_file_name_func=get_cache_filename,
     )
 
+    metadata = marshal_axes(image, len(paths))
+
+    write_multiscales_metadata(parent, paths, **metadata)
```

Comment:
Now that I see this API (introduced in https://github.com/ome/ome-zarr-py/pull/124) in action, I am slightly confused. I think this is primarily because `transformations` is generated from the knowledge of `paths` above and then later re-zipped with `paths` when writing the metadata.

An alternative workflow would be to support `datasets` either as lists of string or dictionaries in  `write_multiscales_metadata`.

Reply:
Perhaps slightly related to my thoughts in https://github.com/ome/ome-zarr-py/pull/161#pullrequestreview-862102250

Reply:
since `datasets` would include `paths`, we would just have `write_multiscales_metadata(group, datasets)` and remove support for paths? Or if `datasets` is a list of strings, treat it as `paths` (wouldn't break the API, but could be more confusing).

Reply:
Similar questions and concerns around the usability came up in https://github.com/ome/ome-zarr-py/pull/157 when adding support for `plate.wells` as list of dictionaries alongside list of strings (also corresponding to individual well paths).

At least, I find `List[Union[str, dict]]` to be a fairly good compromise that allows to handle both the simple use case where a minimal valid spec is generated from a list of paths and the extensible scenario where the caller wants to store additional  metadata (and not necessarily specified by the OME-NGFF spec) to the `datasets` element.

Reply:
See https://github.com/ome/ome-zarr-py/pull/162